deny.toml

```diff
@@ -100,6 +100,7 @@ exceptions = [
   # Each entry is the crate and version constraint, and its specific allow
   # list
   { allow = ["Unicode-DFS-2016"], crate = "unicode-ident" },
+  { allow = ["CC0-1.0"], crate = "tiny-keccak" },
```

Comment:
How did you determine that this allow was valid?

Reply:
Because of [this](https://choosealicense.com/licenses/cc0-1.0/) explanation of the CC0-1.0 license and the fact it's a dependency of cairo crates.